### PR TITLE
initial circleci config

### PR DESCRIPTION
**Pull request by jwdashel**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    <<: *defaults
+
+    steps:
+      - checkout
+      - <<: *restore_node
+      - run:
+          name: Test
+          command: npx ember test
+
+  test-static-cypress:
```
Comment (jjnypr) [inline circle.yml]: I think you can comment this out for now – Cypress isn't set up for this yet

**Comment by jjnypr** [comment]
@jwdashel, you'll need to set up the appropriate environment variables in Circle for the circle.yml file to work properly. Bit of a pain but usually just needs to be set up once. You'll need `PROD_` and `DEMO_` versions of any env vars you reference in there that Circle doesn't give you by default (you can see defaults under any build step in CircleCI by opening the "Spin Up Environment" step e.g. the one here https://circleci.com/gh/nypublicradio/nypr-midterms/2213). You probably also need to set a couple consumed by the Ember application, probably both demo/production versions as well, including `(DEMO|PROD)_SENTRY_DSN`, `(DEMO|PROD)_HOST_WHITELIST`, etc. @kedmundson might have some insight into which ones you need.

File: circle.yml (hunk 1 above)
Comment (jwdashel) [reply circle.yml]: Do I comment out the whole `save_cache` step starting line 78?

Comment (jjnypr) [reply circle.yml]: Just comment out or remove lines 94-105

**Pull request merged**